src/linux_mcp_server/config.py

```diff
@@ -0,0 +1,11 @@
+from pydantic_settings import BaseSettings
+from pydantic_settings import SettingsConfigDict
+
+
+class Config(BaseSettings):
+    model_config = SettingsConfigDict(env_prefix="LINUX_MCP_", env_ignore_empty=True)
+
+    user: str = ""
```

Comment:
Does this somehow automagically get assigned the value of the environment variable (in pseudocode) `$env_prefix + user.to_upper()` as part of the Pydantic magickery?

Reply:
Other way around: this [attribute is assigned](https://docs.pydantic.dev/latest/concepts/pydantic_settings/#environment-variable-names) the value of `LINUX_MCP_USER` by doing a case-insensitive match.